src/coreclr/tools/aot/ObjWriter/build.cmd

```diff
@@ -46,10 +46,7 @@ git apply "%ScriptDir%llvm.patch" || goto Error
 
 :PatchApplied
 
-:: Set CMakePath by evaluating the output from set-cmake-path.ps1
-call "%RepoRoot%src\coreclr\setup_vs_tools.cmd" || goto Error
-for /f "delims=" %%a in ('powershell -NoProfile -ExecutionPolicy ByPass "& ""%RepoRoot%eng\native\set-cmake-path.ps1"""') do %%a
-echo Using CMake at "%CMakePath%"
+call "%RepoRoot%eng\native\init-vs-env.cmd" || goto Error
```

Comment:
I think we should keep the `for /f` and `echo` lines. When `init-vs-env.cmd` is called without arguments, it does not set `CMakePath`.